content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Istio CNI Install Options
+description: Instructions for installing and using Istio with the Istio CNI plugin.
+weight: 70
+keywords: [kubernetes,cni]
+---
+
+The Istio CNI plugin implements the networking details for injecting the Istio sidecar proxy between application pods
+and the container networking infrastructure.  Starting with Istio 1.1, Using the Istio CNI plugin removes the
+requirement for the privileged, `NET_ADMIN` container in the Istio users' application pods.  The
+[Istio CNI plugin](https://github.com/istio/cni) replaces the functionality provided by the `istio-init`
+container injected into application pods' `initContainers` list.
+
+## Prerequisites
+
+1.  Kubernetes installed with the container runtime supporting CNI and `kubelets` configured
+    with the main CNI plugin enabled--`network-plugin=cni`.
```

Comment:
the "--" doesn't look right.  Did you want it to look like a command line option (then need a space before) or as a separator at end of text (which isn't consistent with the rest of the list.